test/io.jl

```diff
@@ -282,6 +282,25 @@ module TestIO
     @test df[:c5][2] == "true"
     @test df[:c5][3] == "true"
 
+    # Readtable column preselection
+    filename = "$data/typeinference/standardtypes.csv"
+    df = readtable(filename)
+    @test size(df) == (3,5)
+    df = readtable(filename, usecols=[1,3,5])
```

Comment:
done.
columns are now sanitized (no duplicates and in order of original columns).
this also mimics the pandas-behavior.

missing tests added.